pkg/featuregates/featuregates.go

```diff
@@ -5,6 +5,12 @@ import (
 	"github.com/openshift/api/features"
 )
 
+// StubOpenShiftVersion is the default OpenShift version placeholder for the purpose of determining
+// enabled and disabled CVO feature gates. It is assumed to never conflict with a real OpenShift
+// version. Both DefaultCvoGates and CvoGatesFromFeatureGate should return a safe conservative
+// default set of enabled and disabled gates, typically with unknownVersion set to true.
+const StubOpenShiftVersion = "0.0.1-snapshot"
```

Comment:
Me, thinking aloud. Totally fine if you ignore.
I feel this thing looks like a constant from `o/lib` as it becomes shared at least among oc and cvo.
I know it might not be worth the effort.

Reply:
I do not think this is _actually_ shared though. The value is the same (mostly for convenience) but it is used differently and it is used exclusively by the `pkg/featuregates/` code as a "if you don't know your version then use this and I will give you some defaults". I was actually considering making the value literally `"<unknown>" but then I chose to be consistent because I still had a comment like "this is used in a similar way as 0.0.1-snapshot in second lever operators".

Reply:
Will do!